src/common/context/OnboardContext.tsx

```diff
@@ -80,6 +90,21 @@ export type OnboardDispatch = Dispatch<Action>;
 type WithDelegatedProps = {
   [k: string]: unknown;
 };
+
+const events = new Events();
+const subscriptions = {
+  address: (addr: string | null) => {
+    events.emit("event", "address", addr);
+  },
+  network: async (networkId: any) => {
+    events.emit("event", "network", networkId);
+  },
+  wallet: async (wallet: Wallet) => {
+    events.emit("event", "wallet", wallet);
+  },
+};
+const onboardInstance = createOnboardInstance(1 as any, subscriptions);
```

Comment:
instanciate single instance of onboard outside of a react component. subscribe only to events inside react component